pytorch_lightning/trainer/trainer.py

```diff
@@ -1184,8 +1177,7 @@ def _call_setup_hook(self) -> None:
 
         if self.datamodule is not None:
             self.datamodule.setup(stage=fn)
-        self.setup(stage=fn)
-        self.lightning_module.setup(stage=fn)
+        self.call_hook("setup", stage=fn)
```

Comment:
Could this change the current workflow ? I believe not, but just to raise attention.

Reply:
It should be good, the order is the same.